core/templates/dev/head/pages/exploration_editor/feedback_tab/feedback_tab.html

```diff
@@ -33,7 +33,7 @@ <h2>Exploration Feedback</h2>
     <div class="row">
       <div class="col-lg-7 col-md-7 col-sm-7">
         <span class="oppia-back-arrow">
-          <button type="button" class="btn btn-default btn-xs protractor-test-oppia-feedback-back-button" ng-click="clearActiveThread()">
+          <button type="button" class="btn btn-default btn-xs protractor-test-oppia-feedback-back-button" ng-click="clearActiveThread(); fetchThreadsAgain()">
```

Comment:
We don't use multiple conditions in ng-click. That's putting too much logic in the HTML.

Please create a new function in the JS along the lines of onBackButtonClicked() or similar, and call that instead.

Reply:
Done